src/internet_identity/src/state.rs

```diff
@@ -12,22 +12,26 @@ use candid::{CandidType, Deserialize};
 use canister_sig_util::signature_map::SignatureMap;
 use ic_cdk::api::time;
 use ic_cdk::trap;
-use ic_stable_structures::storable::Bound;
-use ic_stable_structures::{DefaultMemoryImpl, Storable};
+use ic_stable_structures::DefaultMemoryImpl;
 use internet_identity_interface::internet_identity::types::*;
-use std::borrow::Cow;
 use std::cell::{Cell, RefCell};
 use std::collections::HashMap;
 use std::ops::{Deref, DerefMut};
 use std::time::Duration;
 
 mod temp_keys;
 
-// Default value for max number of delegation origins to store in the list of latest used delegation origins
-const MAX_NUM_DELEGATION_ORIGINS: u64 = 1000;
+/// Default value for max number of delegation origins to store in the list of latest used delegation origins
+pub const DEFAULT_MAX_NUM_DELEGATION_ORIGINS: u64 = 1000;
```

Comment:
nit: there is a slight inconsistency: here we have `DEFAULT_MAX_NUM_..._ORIGINS`, and below we have `DEFAULT_MAX_..._CAPTCHAS` (i.e. without `NUM`), maybe remove the inconsistency? (either way)